src/metrics/documentation_updated/__tests__/documentation-updated.test.ts

```diff
@@ -0,0 +1,113 @@
+import {
+  MetricData,
+  DataEventSignature,
+  MetricsSignature,
+} from "../../../interfaces";
+import { handler } from "../../../handler";
+import mockedPrMerged from "./fixtures/mocked-pull-request-merged.json";
+import { getWebhookEventFixtureList } from "../../../__tests__/fixtures/github-webhook-events";
+
+const octokitResponse = {};
```

Comment:
I only now noticed that the tests always mock the response to the github request as `{}` which means we only test the catch case of the code where the fetch failed and mdFilesChanged is always `0`.

We could set a mocked response here, see the `tooling_usage` or `workflows` tests for examples (the gist is that we write `let octokitResponse = {};` instead and can change the mocked response per test)

e.g.:
```
[
  {
    "filename": "file1.txt",
    "status": "added"
  },
  {
    "filename": "file2.md",
    "status": "removed"
  },
  {
    "filename": "file3.md",
    "status": "modified"
  },
  {
    "filename": "doc/file4.md",
    "status": "added"
  },
  {
    "filename": "docs/file5.md",
    "status": "renamed"
  }
]
```

The above example should probably count two files, just the ".md" files with status `"modified"|"added"|"changed"` (i have no idea what the difference between modified and changed is)

Reply:
I added your suggestion. Currently we don't care about the status, so it's good to have a variety in this in case we need it later on.